src/sentry/integrations/jira/urls.py

```diff
@@ -13,7 +13,7 @@
     url(r'^configure/$', JiraConfigureView.as_view()),
     url(r'^descriptor/$', JiraDescriptorEndpoint.as_view()),
     url(r'^installed/$', JiraInstalledEndpoint.as_view()),
-    url(r'^search/(?P<organization_slug>[^\/]+)/(?P<integration_id>[^\/]+)/$',
+    url(r'^search/(?P<organization_slug>[^\/]+)/(?P<integration_id>\d+)/$',
```

Comment:
I didn't fully understand this, can you give some context about why this is changing?

Reply:
since the integration id refers to the numerical id, rather than the slug, i updated this to reflect that (and also make it the same as what we do in other places: https://github.com/getsentry/sentry/blob/master/src/sentry/api/urls.py#L971)

it doesn't really have anything to do with this change, sorry for lumping it in here